CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 2.5.0
```

Comment:
it looks like it's more like a patch, as we just say, that mapper will also look for iban, not like minor changes.

What do you think?

Reply:
I think it's minor because we are adding some functionality. Previously no ibans. Now ibans returned. I think it's better to increase minor. Let's leave as it is